go-metro/manifest.json

```diff
@@ -14,7 +13,7 @@
   "public_title": "Datadog-Go-Metro Integration",
   "categories":["languages", "log collection"],
   "type":"check",
-  "doc_link": "https://docs.datadoghq.com/integrations/go-metro/",
+  "metric_prefix": "go-metro.", // is this valid see metadata file?
```

Comment:
Metric file seems incorrect. keeping it to `go-metro.`, no dashboards currently exist